src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx

```diff
@@ -1,13 +1,193 @@
+
+  const query = useQuery();
+  const history = useHistory();
+  const location = useStateLocation();
+  const tab = (query.get('tab') || '').toLowerCase();
+  const preselectedTab = TABS.indexOf(tab) === -1 ? 'withdrawal' : tab;
+  const [selectedTab, setSelectedTab] = useState(preselectedTab);
+
+  const [withdrawal, setWithdrawal] = useState<NurseryWithdrawal | undefined>(undefined);
+  const [delivery, setDelivery] = useState<Delivery | undefined>(undefined);
+  const [batches, setBatches] = useState<Batch[] | undefined>(undefined);
+  useEffect(() => {
+    const updateWithdrawal = async () => {
+      const withdrawalResponse = await getNurseryWithdrawal(withdrawalId);
+      if (!withdrawalResponse.requestSucceeded || withdrawalResponse.error) {
+        setWithdrawal(undefined);
+        setDelivery(undefined);
+        setBatches(undefined);
+        snackbar.toastError(`An Error Occurred: ${withdrawalResponse.error}`);
+      } else {
+        setWithdrawal(withdrawalResponse.withdrawal);
+        setDelivery(withdrawalResponse.delivery);
+        setBatches(withdrawalResponse.batches);
+      }
+    };
+
+    updateWithdrawal();
+  }, [withdrawalId, snackbar]);
+
+  useEffect(() => {
+    setSelectedTab((query.get('tab') || 'withdrawal') as string);
+  }, [query]);
+
+  const tabStyles = {
+    fontSize: '14px',
+    padding: theme.spacing(1, 2),
+    minHeight: theme.spacing(4.5),
+    textTransform: 'capitalize',
+    '&.Mui-selected': {
+      color: theme.palette.TwClrTxtBrand as string,
+      fontWeight: 500,
+    },
+  };
+
+  const contentPanelProps = {
+    borderRadius: '32px',
+    backgroundColor: theme.palette.TwClrBg,
+    padding: theme.spacing(3),
+  };
+
+  const handleTabChange = (newValue: string) => {
+    query.set('tab', newValue);
+    history.push(getLocation(location.pathname, location, query.toString()));
+  };
 
-  return <TfMain>Nursery Withdrawal Details {withdrawalId} </TfMain>;
+  return (
+    <TfMain>
+      <PageHeaderWrapper nextElement={contentRef.current} nextElementInitialMargin={-24}>
+        <Box marginBottom={theme.spacing(4)}>
+          <Box>
+            <Link id='back' to={APP_PATHS.NURSERY_WITHDRAWALS} className={classes.backToWithdrawals}>
+              <Icon name='caretLeft' className={classes.backIcon} size='small' />
+              {strings.WITHDRAWAL_LOG}
+            </Link>
+          </Box>
+          <Box
+            padding={isMobile ? theme.spacing(3, 0, 4, 0) : theme.spacing(3, 3, 4, 3)}
+            display='flex'
+            justifyContent='space-between'
+            alignItems='center'
+          >
+            <Typography color={theme.palette.TwClrTxt} fontSize='20px' fontWeight={600} fontStyle='italic'>
+              {withdrawal?.withdrawnDate}
+            </Typography>
+            {withdrawal?.purpose === 'Out Plant' && (
+              <Button size='medium' priority='secondary' onClick={() => undefined} label={strings.REASSIGN} />
```

Comment:
This button should be disabled if this withdrawal has already had reassignments

Reply:
I updated the WithdrawalLogRenderer https://github.com/terraware/terraware-web/blob/main/src/components/NurseryWithdrawals/WithdrawalLogRenderer.tsx#L71 to also not show this if the withdrawal did not have a plot to begin with. (cannot reassign if it wasn't belonging to a plot)